pymc/smc/smc.py

```diff
@@ -175,12 +177,18 @@ def __init__(
 
     def initialize_population(self) -> Dict[str, np.ndarray]:
         """Create an initial population from the prior distribution"""
-        result = sample_prior_predictive(
-            self.draws,
-            var_names=[v.name for v in self.model.unobserved_value_vars],
-            model=self.model,
-            return_inferencedata=False,
-        )
+        sys.stdout.write(" ")
```

Comment:
This makes it show up?

Reply:
yes

Reply:
Haha, I have to test it locally!!! If it works, we should add a comment.

Reply:
It now works on Jupyter. Still does not work in PyCharm, but I think that is their problem

Reply:
Feel free to add more details to this issue https://github.com/pymc-devs/pymc/issues/5828